shuttle/shuttle_controller.py

```diff
@@ -1,9 +1,13 @@
 from flask import session as flask_session, abort
+import re
```

Comment:
Make sure to split up imports into three different sections.

The first section should be global packages (packages that come with python like datetime, json, etc)

The second section should be packages from our virtual environment (things in requirements.txt)

The third section should be stuff from our project (anything from shuttle...)

These three sections should have a space between each section.